### DISCUSSION TITLE
Return type of Get_pane_direction?

### DISCUSSION THREAD

**Discussion by jamesonBradfield** [wezterm/wezterm#3910] [Q&A] [answered]
I asked about a way I could use the same keybinds for switching panes in wezterm, as my Tiling window manager. My thought process was just a simple if statement if I am on the leftMost/RightMost pane, and I was given a couple functions. I am not sure the return type of `wezterm.window:active_tab():get_pane_direction('Left')` I am assuming it's returning tabs_with_info for window.active_tab(), and pane_id for get_pane_direction(''). For the life of me I have no idea how I would approach debugging this(new to lua).

**Comment by bew** [comment, marked as answer]
The documention page of `get_pane_direction` says at the beginning that it returns a pane object.

Although there is no direct link to it, you can find its documentation page in the sidebar on the left:
https://wezfurlong.org/wezterm/config/lua/pane/index.html
